Many performance & flow control updates, and compatible now with Big Sur, fixes #4

mOOky8D opened the pull request:
- Updated the install instructions with some additional formatting and alternate install method.
- Added a pre-built Application workflow to use after the reboot back into macOS to close the SysPref panel.

Commit:
Update README.md

Added some notes on alternative or additional installation steps, and added references to a second automator workflow to clean up after the reboot.

Commit:
Automator workflow to quit System Preferences

Pre-built workflow saved as an Application.  

Quits the System Preferences application if it is running.  This is left open when the Restart to Bootcamp workflow completes (because the system reboots before you can close the app).  Adding this to your Login Items just makes the world a cleaner place.

You can open it with Automator to verify what it does (and doesn't) do.

k-willowhawk committed:
Updates for Big Sur and better flow control

k-willowhawk committed:
Merge branch 'master' of https://github.com/mOOky8D/Restart-to-Bootcamp

Commit:
Merge branch 'master' into master

ghost commented:
- Improved timing and wait conditions
- Fixed UI element path for Big Sur
- Implemented mouse helper command line tool

k-willowhawk committed:
Added selector fix from krazzer

Incorporated community updates from krazzer for better bootcamp disk targeting.
fixes #4

k-willowhawk committed:
Merge branch 'master' of https://github.com/mOOky8D/Restart-to-Bootcamp

k-willowhawk committed:
Merge branch 'master' of https://github.com/mOOky8D/Restart-to-Bootcamp

k-willowhawk committed:
removed no longer used theGRP variable

k-willowhawk committed:
fixed variable for target disk label

ryanmcginnis commented:
thanks a lot!
